### DISCUSSION TITLE
Tooltips

### DISCUSSION THREAD

**Discussion by GWRon** [TVTower/TVTower#575] [Ideas]
Sollten wir Tooltips aus dem Interface so zusammenfuehren, dass immer nur einer gleichzeitig angezeigt wird? Statt jedem Block einen eigenen Tooltip mitzugeben, wuerde es nur einen "globalen" geben - und dessen Inhalt entsprechend variieren.

Gern koennen wir auch ueber eine Art Animation diskutieren - also dass ein Tooltip von Position zu Position "huscht" und dabei seine Groesse anpasst usw.

**Comment by nittka** [comment]
Ja, das klingt nach einer guten Idee. Mehrere sich überlappende Tooltips braucht man nicht im Interface.
Das Verhalten der aufgeschlüsselten Zuschauerzahlen muss man sich dabei aber genau anschauen. Wenn die nämlich dauerhaft einblendbar sind, ist das kein "normaler" Tooltip.
